backend/templates/invite_institution_email.html

```diff
@@ -21,17 +21,17 @@
                     </tr>
                 </table>
 
-                <table width="100%" border="0" cellpadding="0" cellspacing="0" align="center">
+                <table width="100%" height="100%" border="0" cellpadding="0" cellspacing="0" align="center">
```

Comment:
We have to be careful when expressing height in percentage, once, some browsers not allow that unit to style HTML. You should use another type of measurement or test the style with other browsers.